Sources/Rendering/Core/RenderWindowInteractor/index.js

```diff
@@ -388,6 +388,9 @@ function vtkRenderWindowInteractor(publicAPI, model) {
   };
 
   publicAPI.returnFromVRAnimation = () => {
+    if (animationRequesters.size !== 0) {
+      model.animationRequest = requestAnimationFrame(publicAPI.handleAnimation);
+    }
```

Comment:
I would call it AFTER model.vrAnimation is set to false
and do model.FrameTime = -1 prior to request animation frame